axelrod/tests/unit/test_player.py

```diff
@@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
 
     name = "Player"
```

Comment:
We should - but that's one of the four additional tasks I described in the gitter channel.

I think each strategy test needs a ```expected_behaviour``` dictionary with all dimensons specified.